operator/v1/types_console.go

```diff
@@ -202,6 +215,32 @@ type AddPage struct {
 	DisabledActions []string `json:"disabledActions,omitempty"`
 }
 
+type PerspectiveState string
+
+// These are valid perspective states. "Enabled" means the perspective is shown.
+// "Disabled" means the Perspective is hidden.
+const (
+	Enabled  PerspectiveState = "True"
+	Disabled PerspectiveState = "False"
+)
+
+type Perspectives struct {
+	// id defines the id of perspective to disable.
+	// Incorrect or unknown ids will be ignored.
+	// +kubebuilder:validation:Required
+	// +required
+	ID string `json:"id"`
+	// requiresAccessReview defines a list of permission checks.
+	// The perspective will only be shown when all checks are successful.
+	// +kubebuilder:validation:Optional
+	// +optional
+	RequiresAccessReview []AccessReviewResources `json:"requiresAccessReview,omitempty"`
+	// state defines the perspective is enabled or disabled.
+	// +kubebuilder:validation:Optional
+	// +optional
+	State PerspectiveState `json:"state,omitempty"`
```

Comment:
This needs an enum tag
```suggestion
	// +kubebuilder:validation:Enum:=Enabled;Disabled
	State PerspectiveState `json:"state,omitempty"`
```